src/webgpu/web_platform/canvas/configure.spec.ts

```diff
@@ -459,8 +462,9 @@ g.test('viewFormats')
     }, !compatible);
 
     const configuration = ctx.getConfiguration();
-    t.expect(configuration.viewFormats?.length === 1);
-    t.expect(configuration.viewFormats[0] === viewFormat);
+    const viewFormats = Array.from(configuration!.viewFormats!);
```

Comment:
non-blocking aside: Hm, we should actually change this in the types. The `GPUCanvasConfiguration` object that comes from `getConfiguration()` is guaranteed to have an array here (not any other kind of iterable). I'll make a PR.

Reply:
Once https://github.com/gpuweb/types/pull/157 gets merged, I'll go fix the CTS. Thanks!

Reply:
@kainino0x Here's https://github.com/gpuweb/cts/pull/3977